internal/profile/_static/Dockerfile.package-registry

```diff
@@ -5,6 +5,7 @@ ARG PROFILE
 ENV EPR_DISABLE_PACKAGE_VALIDATION=true
 
 ENV EPR_FEATURE_PROXY_MODE=true
+ENV EPR_PROXY_TO=https://epr.elastic.co
```

Comment:
Is it needed if we are going to change the default in package-registry itself?

Reply:
I just add it in case we want to push this change quick into elastic-package.

This is not totally mandatory, if we can wait until the package-registry release and the required update here in elastic-package (update the docker image used for package-registry - https://github.com/elastic/elastic-package/blob/main/internal/profile/static.go#L126).

I'll keep the PR opened in case we see it is worthy.